config/tools-log4j2.yaml

```diff
@@ -12,10 +12,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+Configuration:
```

Comment:
Could you please open a jira for this change? `tools-log4j.properties` is still used in code base

Reply:
Sure.
https://issues.apache.org/jira/browse/KAFKA-18347